frontend/src/components/ACLPage/List/List.tsx

```diff
@@ -36,6 +41,11 @@ const ACList: React.FC = () => {
   } = useBoolean();
   const [rowId, setRowId] = React.useState('');
 
+  // Set the search params to the url based on the localStorage value
+  useEffect(() => {
+    setSearch(searchParams.get('q') || '');
```

Comment:
could you please elaborate on what's happening here? I don't quite grasp the relation to local storage here